File: java/PublicAPIClient/src/com/sdl/GraphQLResponse.java
package com.sdl;

import java.util.List;

public class GraphQLResponse implements IGraphQLResponse{

    private Object Data;
    private List<GraphQLError> Errors;

    @Override
    public Object getData() {
        return Data;
    }

    @Override
    public void setData(Object data) {
        Data = data;
    }

    @Override
    public List<GraphQLError> getErrors() {
        return Errors;
    }

    @Override
    public void setErrors(List<GraphQLError> errors) {
        Errors = errors;
    }
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Response/GraphQLResponse.cs
using System.Collections.Generic;
using Sdl.Tridion.Api.Http.Client;

namespace Sdl.Tridion.Api.GraphQL.Client.Response
{
    /// <summary>
    /// Represents the GraphQL respinse
    /// </summary>
    public class GraphQLResponse : IGraphQLResponse
    {
        /// <summary>
        /// GraphQL Data
        /// </summary>
        public dynamic Data { get; set; }

        /// <summary>
        /// GraphQL Errors
        /// </summary>
        public List<GraphQLError> Errors { get; set; }

        /// <summary>
        /// Response headers
        /// </summary>
        public HttpHeaders Headers { get; set; }
    }
}
